Header header
string object_name
float64 obj_x
float64 obj_y
float64 distance